app/konnect/org-management/teams-reference.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Teams Reference
+no_version: true
+---
+
+All new and existing organizations in Konnect have predefined default teams.
+The default teams can't be modified or deleted.
+
+Keywords:
+* **Fully manage**: Create, read, update, and delete
+* **Partially manage**: Depends on the team
+
+| Team                | Permissions  |
+|---------------------|--------------|
+| Organization Admin  | Users can fully manage all objects, users, and roles in the organization. |
+| Portal Admin        | Users can fully manage all developer portal content, which includes Konnect service pages and supporting content, as well as Dev Portal configuration and Connections. |
+| Portal Viewer       | Users can access the Dev Portal, create applications, and register those applications with services. |
```

Comment:
`Users can access the Dev Portal` - This implies that they can view the rendered dev portal? What I _think_ we're trying to say, is that they can view Dev Portal configuration within Konnect

Reply:
Nice catch - this one is definitely wrongly described in the UI (both in prod and in cloud-02). This description is straight from there. @haydenlam FYI, this is something that will need fixing in the UI.

The kauth definition implies that you can all all developers and application configs, but these permissions are all clearly for Konnect, not for the Dev Portal site.

Rephrasing:
> Users can view Dev Portal configuration in Konnect, including all developers and applications registered through the Dev Portal.

Reply:
Thanks, I will have a look and review these definitions.
Interestingly, this was pulled directly from production RBAC roles.
![image](https://user-images.githubusercontent.com/1035820/160713040-cd180f7d-ed9d-4393-9609-d933a0573250.png)